modules/Users/models/Module.php

```diff
@@ -102,6 +102,8 @@ public function saveLoginHistory($userName, $status = 'Signed in')
 				'logout_time' => null,
 				'status' => $status,
 				'browser' => $browser->name . ' ' . $browser->ver,
+				'userid' => \App\User::getUserIdByName($userName) ?? false,
```

Comment:
```suggestion
				'userid' => \App\User::getUserIdByName($userName),
```
and chenge in getUserIdByName(): ?int